src/pages/workspace/taxes/WorkspaceEditTaxPage.tsx

```diff
@@ -0,0 +1,155 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useMemo, useState} from 'react';
+import {View} from 'react-native';
+import ConfirmModal from '@components/ConfirmModal';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Expensicons from '@components/Icon/Expensicons';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Switch from '@components/Switch';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import {clearTaxRateFieldError, deletePolicyTaxes, setPolicyTaxesEnabled} from '@libs/actions/TaxRate';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import NotFoundPage from '@pages/ErrorPage/NotFoundPage';
+import AdminPolicyAccessOrNotFoundWrapper from '@pages/workspace/AdminPolicyAccessOrNotFoundWrapper';
+import PaidPolicyAccessOrNotFoundWrapper from '@pages/workspace/PaidPolicyAccessOrNotFoundWrapper';
+import type {WithPolicyAndFullscreenLoadingProps} from '@pages/workspace/withPolicyAndFullscreenLoading';
+import withPolicyAndFullscreenLoading from '@pages/workspace/withPolicyAndFullscreenLoading';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+
+type WorkspaceEditTaxPageBaseProps = WithPolicyAndFullscreenLoadingProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.TAXES_EDIT>;
+
+function WorkspaceEditTaxPage({
+    route: {
+        params: {policyID, taxID},
+    },
+    policy,
+}: WorkspaceEditTaxPageBaseProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const currentTaxRate = PolicyUtils.getTaxByID(policy, taxID);
+    const {windowWidth} = useWindowDimensions();
+    const [isDeleteModalVisible, setIsDeleteModalVisible] = useState(false);
+
+    const toggle = () => {
+        if (!policy?.id || !currentTaxRate) {
+            return;
+        }
+        setPolicyTaxesEnabled(policy.id, [taxID], !!currentTaxRate?.isDisabled);
+    };
+
+    const deleteTax = () => {
+        if (!policy?.id) {
+            return;
+        }
+        deletePolicyTaxes(policy?.id, [taxID]);
```

Comment:
i think we agreed to use `route.params.policyID` in each place - check all files